.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     docker:
       - image: cimg/python:3.7.12-node
     resource_class: medium
-    parallelism: 2
+    parallelism: 4
```

Comment:
If the total execution time for `parallelism: 4` is `8*4=32` minutes, aren't we paying more credits than with `parallelism: 2` (`13*2=26` minutes)

Reply:
Yes, we play a little bit more for double the speed, not the double pay, as for 18 min there are ~3 in setup. There is no free lunch, but this is close as it gets. And with more tests coming we had no worry for some time.